src/lib.rs

```diff
@@ -36,6 +37,18 @@ enum Color {
 }
 
 impl Color {
+    fn all() -> &'static [Color] {
```

Comment:
`&'static` 😳

Reply:
Read as `returns a reference to an immutable array of Colors whose lifetime is the lifetime of this program, it will never be dropped'

https://doc.rust-lang.org/stable/rust-by-example/scope/lifetime/static_lifetime.html